First pass plain language proofreading for intro docs (#147)
canonical/ubuntu-server-documentation: explanation/intro-to/mail-services.md

(introduction-to-mail-services)=
# Introduction to mail servers

> Sending email from one person to another over a network or the Internet requires many systems to work together. First, the email client (**Mail User Agent**) sends the message. Then one or more **Mail Transfer Agents** (MTA) transfers the message. The final MTA sends the message to a **Mail Delivery Agent** (MDA) for delivery to the recipient's inbox. Finally, the recipient's email client retrieves the message with a **POP3** or **IMAP** server. These systems must be configured correctly to successfully deliver a message.

Reviewer: ```suggestion
Sending email from one person to another over a network or the Internet requires many systems to work together. First, the sender's email client (**Mail User Agent**) sends the message. Then one or more **Mail Transfer Agents** (MTA) transfers the message. The final MTA sends the message to a **Mail Delivery Agent** (MDA) for delivery to the recipient's inbox. Finally, the recipient's email client retrieves the message, usually via a **POP3** or **IMAP** server. These systems must all be configured correctly to successfully deliver a message.
```

## Mail User Agent

> ### Thunderbird
> [Thunderbird](https://www.thunderbird.net/) is the default Mail User Agent (email client) used by Ubuntu. It comes pre-installed on all Ubuntu machines from Ubuntu 16.04 LTS (Xenial) onwards.

Reviewer: ```suggestion
### Thunderbird

[Thunderbird](https://www.thunderbird.net/) is the default Mail User Agent (email client) used by Ubuntu. It comes pre-installed on all Ubuntu machines from Ubuntu 16.04 LTS (Xenial) onwards.
```

  
  If you need to install Thunderbird manually, [this short guide](https://snapcraft.io/install/thunderbird/ubuntu) will walk you through the steps. 

## Mail Transfer Agent

> ### Postfix
> [Postfix](https://www.postfix.org/) is the default MTA on Ubuntu. It is fast, secure and has good flexibility for administrators. It is compatible with the [sendmail](https://www.authsmtp.com/sendmail/index.html) MTA. 

Reviewer: ```suggestion
### Postfix

[Postfix](https://www.postfix.org/) is the default MTA on Ubuntu. It is fast, secure and has good flexibility for administrators. It is compatible with the [sendmail](https://www.authsmtp.com/sendmail/index.html) MTA. 
```

Reviewer: I think this change should be reverted, since the wording changes have subtly changed the implications of the sentence without gaining additional clarity. Saying it's the default MTA on Ubuntu implies that Ubuntu produces Postfix (we don't), so the "supported" is more important than you might think :D also, saying "for administrators" implies that one needs to be an admin to use it (you don't).

  
  This guide explains {ref}`how to install and configure Postfix <install-postfix>`, including how to configure SMTP for secure communications.

> ### Exim4
> [Exim4](https://www.exim.org/) has extensive configuration features. It can be installed instead of sendmail or Postfix. 

Reviewer: ```suggestion
### Exim4

[Exim4](https://www.exim.org/) has extensive configuration features. It can be installed instead of sendmail or Postfix. 
```

Reviewer: Again, I would suggest reverting these changes. Not much additional clarity is gained, but some context that might be useful to readers is lost.

Author: Reverted to original Exim4 paragraph.
